src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -253,23 +253,23 @@ s32 Player_ActionHandler_12(Player* this, PlayState* play);
 s32 Player_ActionHandler_13(Player* this, PlayState* play);
 s32 Player_ActionHandler_14(Player* this, PlayState* play);
 
-/* Init Mode functions */
-void Player_InitMode_0(PlayState* play, Player* this);
-void Player_InitMode_1(PlayState* play, Player* this);
-void Player_InitMode_2(PlayState* play, Player* this);
-void Player_InitMode_3(PlayState* play, Player* this);
-void Player_InitMode_4(PlayState* play, Player* this);
-void Player_InitMode_5(PlayState* play, Player* this);
-void Player_InitMode_6(PlayState* play, Player* this);
-void Player_InitMode_7(PlayState* play, Player* this);
-void func_80841744(PlayState* play, Player* this);
-void func_80841744(PlayState* play, Player* this);
-void func_8083ADF0(PlayState* play, Player* this);
-void Player_InitMode_B(PlayState* play, Player* this);
-void Player_InitMode_Telescope(PlayState* play, Player* this);
-void Player_InitMode_D(PlayState* play, Player* this);
-void func_8083ADF0(PlayState* play, Player* this);
-void Player_InitMode_F(PlayState* play, Player* this);
+/* Start Mode functions */
+void Player_StartMode_0(PlayState* play, Player* this);
+void Player_StartMode_1(PlayState* play, Player* this);
+void Player_StartMode_2(PlayState* play, Player* this);
+void Player_StartMode_3(PlayState* play, Player* this);
+void Player_StartMode_4(PlayState* play, Player* this);
+void Player_StartMode_5(PlayState* play, Player* this);
+void Player_StartMode_6(PlayState* play, Player* this);
+void Player_StartMode_7(PlayState* play, Player* this);
+void Player_StartMode_WarpTag(PlayState* play, Player* this);
+void Player_StartMode_WarpTag(PlayState* play, Player* this);
+void Player_StartMode_E(PlayState* play, Player* this);
```

Comment:
```suggestion
```

Looks like these forward declarations were duplicated due to `sStartModeFuncs`, though they don't need to be.